sdk/servicebus/tests.yml

```diff
@@ -12,13 +12,11 @@ jobs:
     parameters:
       ServiceDirectory: servicebus
       EnvVars:
-        SERVICE_BUS_HOSTNAME: $(python-sb-livetest-service-bus-hostname)
-        SERVICE_BUS_SAS_POLICY: $(python-sb-livetest-service-sas-policy)
-        SERVICE_BUS_SAS_KEY: $(python-sb-livetest-service-sas-key)
```

Comment:
Have these tests been updated to use the new env variables already?

Reply:
Yes this was done as part of https://github.com/Azure/azure-sdk-for-python/pull/8003